src/store/metadata/stub/content/editions/domestic_energy_performance_certi_existing_dwellings_by_eng_eff_rating.json

```diff
@@ -0,0 +1,97 @@
+{
+    "@context": "https://staging.idpd.uk/#ns",
+    "@id": "https://staging.idpd.uk/datasets/DLUHC/editions",
```

Comment:
Change this to use the same globally unique identifier as above

Reply:
changed this to `epcs-for-existing-domestic-properties`